TabloidCLI/UserInterfaceManagers/TagManager.cs

```diff
@@ -136,8 +136,14 @@ private void Edit()
         private void Remove()
         {
             Console.WriteLine();
-            throw new NotImplementedException();
+            Tag tagToDelete = Choose("Which tag would you like to remove?");
+            if (tagToDelete != null)
+            {
+                _tagRepository.Delete(tagToDelete.Id);
+            }
 
+            Console.WriteLine($"Your tag #{tagToDelete.Name} has been removed.");
```

Comment:
This line the same as the journal variant. I actually tested with tags when I selected to remove one then I just didn't enter anything I received a NullReferenceException. Object reference not set to an instance of an object.

Reply:
Moved code to inside the if statement so only shows up when delete occurs instead of returning an error.